WordPress-Core/ruleset.xml

```diff
@@ -225,6 +225,21 @@
 	<!-- Covers rule: ... array items, only include a space around the index if it is a variable. -->
 	<rule ref="WordPress.Arrays.ArrayKeySpacingRestrictions"/>
 
+	<!-- Rule: In a switch block, there must be no space before the colon for a case statement. -->
+	<!-- Covered by the PSR2.ControlStructures.SwitchDeclaration sniff. -->
+
+	<!-- Rule: Similarly, there should be no space before the colon on return type declarations. -->
+
+	<!-- Covers rule: Unless otherwise specified, parenthesis should have spaces inside of them. -->
```

Comment:
Typo: parenthesis => parentheses.

Reply:
Fixed.

@pento Could you please fix this in the Handbook ? I copied it straight from there.

Reply:
👍🏻 fixed!